### PR TITLE
Add travis.yml

### PR DESCRIPTION
**Pull request by nathanfranklin**
This addresses part of #13 by setting up the CI.

### CODE REVIEW

**Comment by itziakos** [comment]
I would suggest to use the container based configuration http://docs.travis-ci.com/user/migrating-from-legacy/?utm_source=legacy-notice&utm_medium=banner&utm_campaign=legacy-upgrade

**Comment by nathanfranklin** [comment]
@itziakos ,  thanks for the tip.

**Comment by itziakos** [comment]
impressive is runs in just a minute

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+addons:
+  apt:
+    packages:
+      - qt4-qmake
+      - qt4-dev-tools
+      - libpng-dev
+script:
+  - cd src
+  - qmake aviz.pro
+  - make
```
Comment (itziakos) [inline .travis.yml]: why not use `make -j 3`

**Comment by nathanfranklin** [comment]
Actual tests (and possibly static analysis) will be added in a separate PR.

**Pull request merged**